android/src/main/java/com/learnium/RNDeviceInfo/RNDeviceModule.java

```diff
@@ -189,7 +189,6 @@ public Map<String, Object> getConstants() {
 
     final Map<String, Object> constants = new HashMap<>();
 
-    constants.put("uniqueId", getUniqueIdSync());
```

Comment:
with the constant gone, I expect to see a getUniqueIdSync method that is blocking/sync in the java implementation?

Reply:
`getUniqueIdSync()` was already implemented on line 804.  `getUniqueId(Promise p)` added.